### ISSUE TITLE
Replace "POP HL \ JP next" to "JP pop_tos_next"

### ISSUE THREAD

**Issue by redirectme** [Veltas/zenv#7]
System has several "POP HL \ JP next" fragments. This can be replaced with "JP pop_tos_next". Jumping into center of an operation is a "dirty trick" of course, but it works!  
```
pop_pc_next: 
;drop prefix to change POP IY (FD E1) into POP HL (E1) 
pop_tos_next=$+1 
	POP IY  
	; fall through
```

**Comment by Veltas** [comment]
Very clever, will implement after work.

**Comment by Veltas** [comment]
Implemented in ec8b32a, apologies for delay
